### PR TITLE
Migrate to windows-sys

### PR DESCRIPTION
**Pull request by printfn**
The `winapi` crate is no longer maintained, so I've migrated rustyline to [`windows-sys`](https://github.com/microsoft/windows-rs).

### CODE REVIEW

File: src/tty/windows.rs [hunk 1]
```diff
@@ -181,18 +176,18 @@ fn read_input(handle: HANDLE, max_count: u32) -> Result<KeyEvent> {
             return Ok(KeyEvent(K::UnknownEscSeq, M::NONE));
         }
         // TODO GetNumberOfConsoleInputEvents
-        check(unsafe { consoleapi::ReadConsoleInputW(handle, &mut rec, 1, &mut count) })?;
+        check(unsafe { console::ReadConsoleInputW(handle, &mut rec, 1, &mut count) })?;
         total += count;
 
-        if rec.EventType == wincon::WINDOW_BUFFER_SIZE_EVENT {
+        if u32::from(rec.EventType) == console::WINDOW_BUFFER_SIZE_EVENT {
```
Comment (gwenn) [inline src/tty/windows.rs:182]: Inconsistency in windows-sys API ?

**Comment by gwenn** [comment]
```sh
cargo tree --target 'x86_64-pc-windows-gnu' | grep winapi
# nothing
```

**Comment by gwenn** [comment]
Thanks

**Comment by printfn** [comment]
Hi, I was wondering if you were planning on releasing an update to crates.io? It’d be nice to have recent merges like this one available 😊

**Pull request merged**